public/resources/core/js/api/party/individual.controller.js

```diff
@@ -0,0 +1,211 @@
+/**
```

Comment:
Why have you used a new folder to this file? Cannot you use the `service`s folder you have created for the rest of APIs?